### PR TITLE
Cache

### PR DESCRIPTION
**Pull request by shefnerok**
cache feature for PR

### CODE REVIEW

**Comment by azetilov** [comment]
Please resolve conflicts

**Comment by shefnerok** [comment]
Resolved

File: src/Fileparser.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+			Element roomElement = roomsElement.child(0);
+			// rooms
+			String rooms = roomElement.text();
+
+			roomsList.add(new Rooms(rooms));
+		});
+
+		// CLASS PRICE
+		priceElements.forEach(priceElement -> {
+			Element priceElement1 = priceElement.child(2);
```
Comment (azetilov) [inline src/Fileparser.java:61]: priceInUSD

File: src/Fileparser.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+public class Fileparser {
+
+	public static void start() throws IOException {
+
+		File input = new File("Cache/cache.txt");
+		// windows-1251 - encoding set do display cyrillic chars.
+		Document parsingDocument = Jsoup.parse(input, "windows-1251");
+
+		// data slots
+		ArrayList<Apartment> apartmentList = new ArrayList<>();
```
Comment (azetilov) [inline src/Fileparser.java:23]: Use plurals instead of adding "List" everywhere. E.g. roomsList -> rooms

**Comment by shefnerok** [comment]
accidentally added testing feature into this PR

**Pull request merged**